src/dialog-user/components/dialog-user/dialogUser.ts

```diff
@@ -233,6 +237,18 @@ export class DialogUserController extends DialogClass implements IDialogs {
     }
   }
 
+  /**
+   * Determines if the given field type is a subclass of DialogFieldSortedItem
+   * @memberof DialogUserController
+   * @function isASortedItemDialogField
+   * @param fieldType {string} This is the field type that should be used for comparison
+   */
+  private isASortedItemDialogField(fieldType) {
+    return fieldType === 'DialogFieldDropDownList' ||
+           fieldType === 'DialogFieldRadioButton' ||
+           fieldType === 'DialogFieldTagControl';
```

Comment:
@eclarizio - Is this a list that would have to be manually modified if there is a new subclass of `DialogFieldSortedItem`, something that could be missed in the future?

Reply:
Yes, that's possible. Not exactly sure the best way around that at the moment, unfortunately. Luckily, in the time I've known dialogs, we've never added another type (and it's been 4 years), so, I highly doubt we'll be adding another subclass for sorted items any time soon 😀

Reply:
I can get behind that. 4 years seems appropriate for Subclass additions.